streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -2330,4 +2343,121 @@ public void shouldNotAllowNullNamedOnProcessWithStores() {
         assertThat(exception.getMessage(), equalTo("named can't be null"));
     }
 
+
+    @Test
+    public void shouldKTableFromKStream() {
+        final Consumed<String, String> consumed = Consumed.with(Serdes.String(), Serdes.String())
+            .withName("table-process");
+        final Consumed<String, String> consumed2 = Consumed.with(Serdes.String(), Serdes.String())
+            .withName("stream-table-process");
+
+        final StreamsBuilder builder = new StreamsBuilder();
+
+        final String input = "input";
+        final String streamToTable = "streamToTableInput";
+        final String output = "output";
+        final String streamToTableOutput = "streamToTableOutput";
+
+        builder.table(input, consumed).toStream().to(output);
```

Comment:
Why do we need to create this table in this test? It seems more "straight-forward" to verify the output from `streamToTableOutput` directly with a hard-coded list of expected values?